server.js

```diff
@@ -17,10 +22,19 @@ app.use(express.static(__dirname));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+//Cross-Origin Resource Sharing CORS
+//To enable access of resources from our server
+
+if(process.env.NODE_ENV === 'development'){
```

Comment:
this should be more like `if (process.env.NODE_ENV !== 'production')`. Unless you have something somewhere that is setting the `NODE_ENV` to`development`.